### DISCUSSION TITLE
zh/guide/

### DISCUSSION THREAD

**Discussion by giscus** [ZLMediaKit/docs#12] [Announcements]
# zh/guide/

一个基于 C++11 的高性能运营级流媒体服务框架 ![badge (https://img.shields.io/badge/license-MIT-green.svg)](https://github.com/ZLMediaKit/ZLMediaKit/blob/master/LICENSE) ![badge (https://img.shield...

https://docs.zlmediakit.com/zh/guide/

**Comment by keke9036** [comment]
功能一览众的图里面看不到文字信息
